subsys/nrf_security/src/drivers/cracen/cracenpsa/src/kmu.c

```diff
@@ -342,6 +347,26 @@ psa_status_t convert_from_psa_attributes(const psa_key_attributes_t *key_attr,
 	memset(metadata, 0, sizeof(*metadata));
 	metadata->metadata_version = 0;
 
+	metadata->key_usage_scheme = CRACEN_PSA_GET_KEY_USAGE_SCHEME(psa_get_key_id(key_attr));
+	if (metadata->key_usage_scheme > KMU_METADATA_SCHEME_RAW) {
```

Comment:
It's kind of a nit but maybe it makes sense to have explicit check on the specific keys that we support rather than implicit like this. 
Maybe a switch statement or something which the key types that we support and the default option can return invalid or something.

That's more of a suggestion though, we can also keep it like that if you think that it is better.